crates/database/src/trades.rs

```diff
@@ -43,14 +43,20 @@ LEFT OUTER JOIN LATERAL (
     LIMIT 1
 ) AS settlement ON true
 JOIN orders o
-ON o.uid = t.order_uid
-WHERE
-    o.uid IS NOT null
-AND
-    ($1 IS NULL OR o.owner = $1)
-AND
-    ($2 IS NULL OR o.uid = $2)
-    "#;
+ON o.uid = t.order_uid"#;
+    const QUERY: &str = const_format::concatcp!(
+        COMMON_QUERY,
+        " WHERE o.uid IS NOT null ",
```

Comment:
Is this needed? I thought it was a `PRIMARY KEY` and therefore automatically `NON NULL`.

Reply:
Good catch. I never looked at it too closely, as it is in the original code.